test/rails5.0/app/cells/formtastic_cell.rb

```diff
@@ -1,4 +1,6 @@
 class FormtasticCell < Cell::ViewModel
+  view_paths << "#{Rails.root}/app/cells"
```

Comment:
Adding `view_paths` and including the `Erb` module here must happen automatically as this is a complete integration test how this gem behaves in a real Rails application. Please remove it. :beers: